bin/pycbc_inspiral

```diff
@@ -307,13 +308,16 @@ with ctx:
 
     tsetup = time.time() - tstart
 
+    logging.info("Performing q-transform on analysis segments")
+    out_vals['qtransform'] = qtransform.inspiral_qtransform_generator(segments)
```

Comment:
I.E. does it actually save out the results correctly? Also, we don't always want to have these results. please add a command line arguments to toggle the qtransform related calls. If this code actually functions, you can update the dictionary above and put both these lines into an if statement.